mmseg/models/decode_heads/setr_up_head.py

```diff
@@ -0,0 +1,131 @@
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn import ConvModule, build_norm_layer
+
+from ..builder import HEADS
+from ..utils import trunc_normal_
+from .decode_head import BaseDecodeHead
+
+
+class UPModule(nn.ModuleList):
+
+    def __init__(self,
+                 in_channels,
+                 out_channels=256,
+                 num_convs=1,
+                 up_mode='bilinear',
+                 up_scale=4,
+                 align_corners=False,
+                 num_up_layer=0,
+                 kernel_size=3,
+                 norm_cfg=None,
+                 act_cfg=dict(type='ReLU')):
+        super(UPModule, self).__init__()
+        self.num_convs = num_convs
+        self.up_mode = up_mode
+        self.up_scale = up_scale
+        self.align_corners = align_corners
+        self.num_up_layer = num_up_layer
+        self.kernel_size = kernel_size
+
+        for i in range(num_convs):
+            if i == 0:
+                tmp_in_channels = in_channels
+                tmp_out_channels = out_channels
+            else:
+                tmp_in_channels = out_channels
+                tmp_out_channels = out_channels
```

Comment:
These lines are not necessary. 

We just need 1 variable `last_out_channels` to record the output channels out last conv.